### PR TITLE
Add LIP "Introduce Interoperability module"

### PR DESCRIPTION
**Pull request by ricott1**

### CODE REVIEW

File: proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md [hunk 1]
```diff
@@ -0,0 +1,862 @@
+```
+LIP: <LIP number>
+Title: Properties, serialization, and initial values of interoperability module
+Author: Alessandro Ricottone <alessandro.ricottone@lightcurve.io>
+Type: Standards Track
```
Comment (IkerAlus) [inline proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md:5]: Add the `Discussion-To` field.

File: proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md [hunk 2]
```diff
@@ -0,0 +1,862 @@
...
+### Liveness Condition
+
+Active sidechains are required to prove their liveness to the mainchain at least once every 30 days. This is done by including a CCU on the mainchain. If a sidechain does not post a CCU within 30 days, the sidechain account is terminated.  
+
+This rule guarantees that users do not send funds to inactive sidechains and that users who have tokens in a sidechain which stops communicating with the ecosystem can recover their tokens. The same rule applies to the mainchain account on a sidechain. At least once within 30 days, a mainchain CCU has to be included in the sidechain, otherwise, the mainchain account on the sidechain is terminated.
+
+
+### Life Cycle of a Sidechain
+
+The life cycle of a sidechain can be split into 3 parts, corresponding to the 3 [values of the account status property](#List-of-Account-status-Values): ''registered'', ''active'', and ''terminated''.
```
Comment (IkerAlus) [inline proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md:88]: Seems like this link to the status is broken

**Review by IkerAlus — CHANGES_REQUESTED** [review]
Two small fixes. Also the pseudocode snippets are in `java` style instead of the recommended `python` style, but they are still perfectly clear so not strictly required to update.

Apart from that, LGTM, great work.

**Review by MaximeGagnebin — APPROVED** [review]
Very nice !

File: proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md (hunk 1 above)
Comment (ricott1) [reply proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md:5]: fixed

File: proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md (hunk 2 above)
Comment (ricott1) [reply proposals/lip-properties-serialization-and-initial-values-of-the-interoperability-module.md:88]: yep, we removed the section

**Review by IkerAlus — APPROVED** [review]
LGTM, great work.

**Review by ManuGowda — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]

**Review by IkerAlus — APPROVED** [review]
new changes LGTM.

**Review by MaximeGagnebin — APPROVED** [review]

**Review by MaximeGagnebin — APPROVED** [review]
lgtm

**Pull request merged**